server/text_generation_server/layers/fp8.py

```diff
@@ -450,6 +503,20 @@ def get_shared_device_identity(cls, device):
         return cls._device_identity_cache[device]
 
     def forward(self, input: torch.Tensor) -> torch.Tensor:
+        if self.weight_block_size is not None:
+            qinput, scale = per_token_group_quant_fp8(input, self.weight_block_size[1])
```

Comment:
Would be nice to have a comment here, why `self.weight_block_size[1]`? (can also be a follow-up PR to avoid the build time)

Reply:
From paper, they quantize input as a per tile.
```
at a more granular level. As illustrated in Figure 7 (a), (1) for activations, we group and
scale elements on a 1x128 tile basis (i.e., per token per 128 channels); and (2) for weights, we
group and scale elements on a 128x128 block basis (i.e., per 128 input channels per 128 output
channels). 
```